File: 백준/Bronze/10810. 공 넣기/공 넣기.cc
#include <stdio.h>

int main() {

	int n, m;
	int arr[101] = {0,};
	int i, j, k;

	scanf("%d %d", &n, &m);

	for (int l = 0; l < m; l++)
	{
		scanf("%d %d %d", &i,&j, &k);

		for (int p = i; p <= j; p++)
		{
			arr[p-1] = k;
		}
	}

	for (int i = 0; i < n; i++)
	{
		printf("%d ", arr[i]);
	}
	
	


	return 0;
}

File: 백준/Bronze/10810. 공 넣기/공 넣기.java
import java.util.Scanner;

public class Main {
	public static void main(String[] args) {
		Scanner input = new Scanner(System.in);
		int n = input.nextInt();
		int m = input.nextInt();
		int arr[] = new int[n];
		int i,j,k;
		for (int v = 0; v < m; v++) {
			i = input.nextInt();
			j = input.nextInt();
			k = input.nextInt();
			for (int l = i; l <=j; l++) {
				arr[l-1] = k;
			}
		}
		for (int u = 0; u < n; u++) {
			System.out.print(arr[u]+ " ");
		}
		
		
		
	}
}
